### PR TITLE
documentation: frame renamed to dataset

### PR DESCRIPTION
**Pull request by kadhikari**

### CODE REVIEW

**Comment by kadhikari** [comment]
@prhod 
Could you read and valid this request please

File: documentation/ntfs/ntfs_changelog.md [hunk 1]
```diff
@@ -27,10 +27,15 @@
...
     * Gestion du fichier equipments.txt
-* Version 0.6 du 24/112015
-    * Ajout de la gestion des services (fichier frames.txt) et modification des contributeurs pour ajouter une licence et un site web 
+* Version 0.6 du 24/11/2015
+    * Ajout de la gestion des services (fichier frames.txt) et modification des contributeurs pour ajouter une licence et un site web
     * Ajout de la gestion des données adaptées (dans feed_info.txt une clé "revised_networks" listant les réseaux impactés par une grève ou une autre perturbation)
     * Modification de l'identifiant des calendriers et des calendriers de période
     * Modification des commentaires : Ajout d'un label optionnel et modification du nom du champ "comment_name" en "comment_value"
     * Modification du champ "is_forward" en "direction_type" dans le fichier routes.txt et passage en champ de type chaine
+* Version 0.6 du 25/02/2016
```
Comment (TeXitoi) [inline documentation/ntfs/ntfs_changelog.md]: Version 0.7 peut-être ?

**Comment by kadhikari** [comment]
This correction is valid as voted by Heros.

**Comment by antoine-de** [comment]
OK for me

**Pull request merged**